### PR TITLE
8 main components for views

### PR DESCRIPTION
**Pull request by heliomsf**
Little tweaks that I've been working on since yesterday.

### CODE REVIEW

File: app/assets/stylesheets/config/_fonts.scss [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
 @import url('https://fonts.googleapis.com/css?family=Nunito:400,700|Work+Sans:400,700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Courier+Prime:wght@400;700&family=Kanit:wght@600&family=Playfair+Display:ital,wght@0,400;0,600;0,700;1,400;1,600;1,700&family=Rubik:wght@300;400;600&display=swap');
 
 // Define fonts for body and headers
-$body-font: "Work Sans", "Helvetica", "sans-serif";
-$headers-font: "Nunito", "Helvetica", "sans-serif";
+$body-font: "Rubik", "sans-serif";
+$headers-font: "Playfair Display", "serif";
 
 // To use a font file (.woff) uncomment following lines
```
Comment (rudygasson) [inline app/assets/stylesheets/config/_fonts.scss:9]: Do we need all this outcommented stuff?

File: app/views/books/show.html.erb [hunk 2]
```diff
@@ -1,14 +1,14 @@
-<div class="container p-3 text-center">
-  <div class="book-cover">
-    <img src=<%= @book.cover_image %> alt="book cover" class="mb-3">
+<div class="container p-3">
+  <div class="container book-cover book-info">
+    <img src=<%= @book.cover_image %> alt="book cover" class="mb-3 img-center">
     <h2><%= @book.title %></h2>
     <p>by <%= @book.author %></p>
-    <span>Description:</span>
+    <%# <span>Description:</span> %>
```
Comment (rudygasson) [inline app/views/books/show.html.erb:6]: You can just delete the things. Please do not leave to much clutter. Version control is taking care of it, we can always find it in earlier versions.

Comment (heliomsf) [reply app/views/books/show.html.erb:6]: Ok!

File: app/views/pages/home.html.erb [hunk 3]
```diff
@@ -32,10 +32,16 @@
...
-              <img src="<%= book.thumbnail %>" alt="book card">
+              <div class="book-box">
+                <img src="<%= book.thumbnail %>" alt="book card">
+              </div>
               <div class="card-book-infos">
                 <h2><%= "#{book.title}"%></h2>
                 <p><%= book.author %></p>
+                <br>
+                <%# ##### Calendar and trash buttons ##### %>
+                  <%# <span class="material-symbols-outlined">event_available</span>
```
[the remark this answers is not in the document]
Comment (heliomsf) [reply app/views/pages/home.html.erb:43]: Good idea. I can try to work on that

**Pull request closed without merging**